File: server/add_attribute.js
var args = process.argv.slice(2);

var MongoClient = require('mongodb').MongoClient
  , assert = require('assert');
var fs = require('fs');

// Connection URL
var url = 'mongodb://localhost:27017/pairwise-collector';

// Use connect method to connect to the Server
MongoClient.connect(url, function(err, db) {
  assert.equal(null, err);

  var settings = db.collection('settings');

  attr = args[0];
  console.log("Adding attribute option: " + attr);

  settings.insert({"name" : attr, "type" : "attribute" }, function(e, r) {});

  process.exit(0);
});

File: server/add_attribute.py
# Adds a new attribute to the database
from pymongo import MongoClient
import sys
import os

attr = sys.argv[1]

client = MongoClient('localhost', 27017)
db = client.attributes
settings = db.settings

print("Adding attribute option: " + attr)
settings.insert_one({ "name" : attr, "type" : "attribute" })